include/mbgl/style/position.hpp

```diff
@@ -9,7 +9,7 @@ namespace style {
 class Position {
 public:
     Position() = default;
-    Position(std::array<float, 3>& position_)
+    Position(const std::array<float, 3>& position_)
```

Comment:
I thought I saw there might be a separate PR for this. Just checking you meant to leave this in.